tox.ini

```diff
@@ -7,9 +7,12 @@ envlist =
     py35,
     py36,
     py36-datetime,
+    py37,
```

Comment:
This requires each developer to have Python 3.7 installed as otherwise the `tox` run will fail.
I'm okay to test it on TravisCI but until we do not officially support this version I do not like it to show up here.
(For official support much more has to be done than that the tests run on this version!)

Reply:
ok, we could remove them in tox.ini but should keep them in travis to see if we have potential to expand platforms.